com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -125,28 +131,34 @@ public void build(Task task) throws IOException, CompileExceptionError {
         }
 
         if (getOutputHlslFlag()) {
-            compileOptions.forceIncludeShaderLanguages.add(ShaderDesc.Language.LANGUAGE_HLSL);
+            addUniqueShaderLanguage(ShaderDesc.Language.LANGUAGE_HLSL);
         }
         if (getOutputSpirvFlag()) {
-            compileOptions.forceIncludeShaderLanguages.add(ShaderDesc.Language.LANGUAGE_SPIRV);
+            addUniqueShaderLanguage(ShaderDesc.Language.LANGUAGE_SPIRV);
         }
         if (getOutputWGSLFlag()) {
-            compileOptions.forceIncludeShaderLanguages.add(ShaderDesc.Language.LANGUAGE_WGSL);
+            addUniqueShaderLanguage(ShaderDesc.Language.LANGUAGE_WGSL);
+        }
+        if (getOutputGLSLFlag()) {
```

Comment:
This flag, compared to the specific language flags, adds the GLSL languages that is supported by the platform we are building for, and not just specific languages (which is more for tests).